### PR TITLE
feat(ci): move to github actions

### PR DESCRIPTION
**Pull request by kduret**
## Description

test github actions

### CODE REVIEW

File: packaging/centreon-plugin-Applications-Monitoring-Centreon-Central/rpm.json [hunk 1]
```diff
@@ -7,4 +7,5 @@
         "perl(DBD::mysql)"
     ],
     "custom_pkg_data": "Obsoletes:    centreon-plugin-meta"
+
```
Comment (kduret) [inline packaging/centreon-plugin-Applications-Monitoring-Centreon-Central/rpm.json:10]:
```suggestion
```

File: packaging/centreon-plugin-Cloud-Azure-Network-TrafficManager/pkg.json [hunk 2]
```diff
@@ -8,5 +8,6 @@
         "cloud/azure/management/monitor/mode/discovery.pm",
         "cloud/azure/management/monitor/mode/health.pm",
         "cloud/azure/network/trafficmanager/"
+
```
Comment (kduret) [inline packaging/centreon-plugin-Cloud-Azure-Network-TrafficManager/pkg.json:11]:
```suggestion
```

**Review by sc979 — APPROVED** [review]

File: .github/actions/rpm-delivery/action.yml [hunk 3]
```diff
@@ -83,9 +83,9 @@ runs:
           scp -o StrictHostKeyChecking=no ./*.rpm "${{ inputs.yum_repo_address }}:$TARGET"
 
           # Cleanup is done on unstable repository only
-          if [ "$REPOTYPE" == "unstable" ]; then
-            ssh -o StrictHostKeyChecking=no "${{ inputs.yum_repo_address }}" "ls -drc $PROJECT_LOCATION/* 2>&- | head -n -1 | xargs rm -rf"
-          fi
+          #if [ "$REPOTYPE" == "unstable" ]; then
+          #  ssh -o StrictHostKeyChecking=no "${{ inputs.yum_repo_address }}" "ls -drc $PROJECT_LOCATION/* 2>&- | head -n -1 | xargs rm -rf"
+          #fi
```
Comment (tuntoja) [inline .github/actions/rpm-delivery/action.yml:88]: do we not clean repo anymore ?

File: .github/workflows/docker-builder-packaging-plugins.yml [hunk 4]
```diff
@@ -6,16 +6,16 @@ concurrency:
...
+  push:
+    branches:
+      - develop
+    paths:
+      - ".github/workflows/docker-builder-packaging-plugins.yml"
+      - ".github/docker/*"
+  pull_request:
+    paths:
+      - ".github/workflows/docker-builder-packaging-plugins.yml"
+      - ".github/docker/*"
```
Comment (tuntoja) [inline .github/workflows/docker-builder-packaging-plugins.yml:18]: Should we not remove run triggers on workflow file changes like we are doing on the other repos?

**Review by tuntoja — APPROVED** [review]

File: .github/workflows/docker-builder-packaging-plugins.yml (hunk 4 above)
Comment (kduret) [reply .github/workflows/docker-builder-packaging-plugins.yml:18]: it is the workflow to build packaging docker, so there is no issue for releases

**Pull request merged**